src/EnergyPlus/DaylightingManager.cc

```diff
@@ -2235,22 +2126,29 @@ void FigureDayltgCoeffsAtPointsForWindowElements(
             // Look up the TDD:DOME object
             PipeNum = state.dataSurface->SurfWinTDDPipeNum(IWin);
             // Unshaded visible transmittance of TDD for a single ray from sky/ground element
-            TVISB = TransTDD(state, PipeNum, COSB, DataDaylightingDevices::iRadType::VisibleBeam) * state.dataSurface->SurfWinGlazedFrac(IWin);
+            TVISB = DaylightingDevices::TransTDD(state, PipeNum, COSB, DataDaylightingDevices::iRadType::VisibleBeam) *
+                    state.dataSurface->SurfWinGlazedFrac(IWin);
 
         } else { // Regular window
             if (state.dataSurface->SurfWinWindowModelType(IWin) != WindowBSDFModel) {
                 // Vis trans of glass for COSB incidence angle
-                TVISB = POLYF(COSB, state.dataConstruction->Construct(IConst).TransVisBeamCoef) * state.dataSurface->SurfWinGlazedFrac(IWin) *
-                        state.dataSurface->SurfWinLightWellEff(IWin);
+                TVISB = General::POLYF(COSB, state.dataConstruction->Construct(IConst).TransVisBeamCoef) *
+                        state.dataSurface->SurfWinGlazedFrac(IWin) * state.dataSurface->SurfWinLightWellEff(IWin);
             } else {
                 // Complex fenestration needs to use different equation for visible transmittance.  That will be calculated later
                 // in the code since it depends on different incoming directions.  For now, just put zero to differentiate from
                 // regular windows
                 TVISB = 0.0;
             }
             if (ExtWinType == DataDaylighting::iExtWinType::AdjZoneExtWin) {
+                int zoneNum = 0;
+                if (CalledFrom == DataDaylighting::iCalledFor::RefPoint) {
+                    zoneNum = state.dataDaylightingData->daylightControl(daylightCtrlNum).zoneIndex;
+                } else if (CalledFrom == DataDaylighting::iCalledFor::MapPoint) {
+                    zoneNum = state.dataDaylightingData->IllumMapCalc(MapNum).zoneIndex;
+                }
```

Comment:
OK so 2 possibilities. But this certainly has implications. I suspect this will have diffs in daylighting example files? and if not, can we modify those files to show this issue?